### ISSUE TITLE
Add extra conda commands (delete, list)?

### ISSUE THREAD

**Issue by pllim** [SaraOgaz/STScI-env#15]
Would be nice to just have a small sub-section to show commands on how to:
- [ ] List all environments (`conda info --envs` or `conda env list`)
- [ ] Remove an unwanted environment (`conda env remove -n [env name]`)
- [ ] List all channels (`conda config --get channels`)
- [ ] Remove an unwanted custom channel (`conda config --remove channels [channel name/url]`)

**Comment by SaOgaz** [comment]
i agree, I will add this in when I get back from my trip, good idea, thanks!
